config-overrides.js

```diff
@@ -1,10 +1,19 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 const path = require('path')
-const { override, addBabelPreset, addBabelPlugin, addWebpackAlias, addWebpackModuleRule } = require('customize-cra')
+const StylelintPlugin = require('stylelint-webpack-plugin')
+const {
+  override,
+  addBabelPreset,
+  addBabelPlugin,
+  addWebpackAlias,
+  addWebpackModuleRule,
+  addWebpackPlugin,
+} = require('customize-cra')
 
 module.exports = {
   webpack: override(
     addBabelPlugin('@emotion/babel-plugin'),
+    addWebpackPlugin(new StylelintPlugin({ files: './src/**/*.{tsx,ts}' })),
```

Comment:
I'm wondering whether we need this. I have a feeling this slows down "rebuild" time in development although cannot say for sure, hard to measure for sure. I think editor+CI linting would be enough. What do you think?

Reply:
Editor linting doesn't work without `stylelint` extension.  This can be helpful for people who don't use it. They will see the nice output in the console what's wrong -  for example
![image](https://user-images.githubusercontent.com/51168865/121487215-29691600-c9d2-11eb-963a-dca7cec42b02.png)
As for performance - I didn't spot the difference, but I may be wrong.

Reply:
Okay, let's add that. However, if we move to some other build tool it's quite possible we'll get rid of both `eslint` and `stylelint` plugins